app/src/main/java/de/tum/in/tumcampusapp/component/other/generic/drawer/DrawerMenuHelper.kt

```diff
@@ -94,7 +95,8 @@ class DrawerMenuHelper(private val mContext: Context, private val mDrawerLayout:
                 SideNavigationItem(R.string.news, R.drawable.ic_rss, NewsActivity::class.java, false, false),
                 SideNavigationItem(R.string.barrier_free, R.drawable.ic_accessible, BarrierFreeInfoActivity::class.java, false, false),
                 SideNavigationItem(R.string.opening_hours, R.drawable.ic_time, OpeningHoursListActivity::class.java, false, false),
-                SideNavigationItem(R.string.study_plans, R.drawable.ic_study_plans, CurriculaActivity::class.java, false, false)
+                SideNavigationItem(R.string.study_plans, R.drawable.ic_study_plans, CurriculaActivity::class.java, false, false),
+                SideNavigationItem(R.string.events, R.drawable.ic_time, EventDetailsActivity::class.java, false, false)
```

Comment:
Why do we want this in the menu? We actually only want the list of events in the menu, not the details.

Reply:
Yes, correct! I only added this temporarily to test my code. This should later be removed and the Event Details will be linked from the Event list which is implemented in #812.